modules/ROOT/pages/database-internals/store-formats.adoc

```diff
@@ -24,7 +24,8 @@ Block format means a few pages need to be loaded to serve a query, i.e. fewer pa
 * *Property access:* Properties are stored in blocks with their nodes and relationships drastically reducing the amount of pointer chasing required to access properties.
 * *Entity limits:* Able to run graphs at large scales.
 Supports the highest limits at the time of writing.
-See <<block-format-limits, Block format entity limits>> for details.
+See <<block-format-limits, Block format entity limits>> for details. +
+label:new[Introduced in 5.25] The max length of token names (i.e., label, property key, and relationship type names) is 16,383 characters.
```

Comment:
Should we link to or mention that 16,383 characters is a GQL identifier max length?